# --- T2-COPYRIGHT-NOTE-BEGIN ---
# T2 SDE: package/*/hyprland/sde-version.patch
# Copyright (C) 2023 - 2024 The T2 SDE Project
# 
# This Copyright note is generated by scripts/Create-CopyPatch,
# more information can be found in the files COPYING and README.
# 
# This patch file is dual-licensed. It is available under the license the
# patched project is licensed under, as long as it is an OpenSource license
# as defined at http://www.opensource.org/ (e.g. BSD, X11) or under the terms
# of the GNU General Public License version 2 as used by the T2 SDE.
# --- T2-COPYRIGHT-NOTE-END ---

--- hyprland-source/src/helpers/MiscFunctions.cpp.vanilla	2024-06-12 18:37:29.998000000 +0200
+++ hyprland-source/src/helpers/MiscFunctions.cpp	2024-06-12 18:37:58.065000000 +0200
@@ -663,7 +663,7 @@
     // log etc
     Debug::log(LOG, "os-release:");
 
-    Debug::log(NONE, "{}", execAndGet("cat /etc/os-release"));
+    Debug::log(NONE, "{}", execAndGet("cat /etc/SDE-VERSION"));
 }
 
 void matrixProjection(float mat[9], int w, int h, wl_output_transform tr) {
@@ -934,3 +934,4 @@
     *ro_fd_ptr = ro_fd;
     return true;
 }
+
--- hyprland-source/src/debug/CrashReporter.cpp.vanilla	2024-06-12 18:36:57.883000000 +0200
+++ hyprland-source/src/debug/CrashReporter.cpp	2024-06-12 18:37:21.254000000 +0200
@@ -152,7 +152,7 @@
 #endif
 
     finalCrashReport += "\n\nos-release:\n";
-    finalCrashReport.writeCmdOutput("cat /etc/os-release | sed 's/^/\t/'");
+    finalCrashReport.writeCmdOutput("cat /etc/SDE-VERSION | sed 's/^/\t/'");
 
     // dladdr1()/backtrace_symbols()/this entire section allocates, and hence is NOT async-signal-safe.
     // Make sure that we save the current known crash report information,
